src/components/RefinementList/__tests__/RefinementList-test.js

```diff
@@ -56,8 +56,8 @@ describe('RefinementList', () => {
         </div>
       </div>
     );
-    expect(out.props.children[0].key).toEqual('facet1');
-    expect(out.props.children[1].key).toEqual('facet2');
+    expect(out.props.children[0][0].key).toEqual('facet1');
```

Comment:
It's because we have 2 children, therefore the children is not a single element but an array of elements. https://facebook.github.io/react/tips/children-props-type.html